internal/cli/atlas/clusters/create.go

```diff
@@ -271,6 +299,9 @@ For full control of your deployment, or to create multi-cloud clusters, provide
 	cmd.Flags().BoolVar(&opts.enableTerminationProtection, flag.EnableTerminationProtection, false, usage.EnableTerminationProtection)
 	cmd.Flags().StringToStringVar(&opts.tag, flag.Tag, nil, usage.Tag)
 
+	cmd.Flags().BoolVarP(&opts.EnableWatch, flag.EnableWatch, flag.EnableWatchShort, false, usage.EnableWatch)
+	cmd.Flags().UintVar(&opts.Timeout, flag.WatchTimeout, 0, usage.WatchTimeout)
```

Comment:
use https://pkg.go.dev/github.com/spf13/pflag#Duration it'll save you the cast to duration

Reply:
Make sense. I will address this in [CLOUDP-181597](https://jira.mongodb.org/browse/CLOUDP-181597). Thank You!